### PR TITLE
Add MySQL connection and docker-compose settings

### PR DESCRIPTION
**Pull request by natatkachenko**

### CODE REVIEW

File: Taxopark/DependencyInjection/DependencyContainer.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Text;
+using Taxopark.Infrastructure.Data;
+
+namespace DependencyInjection
+{
+    public static class DependencyContainer
+    {
+        public static void AddDependencies(IServiceCollection services, IConfiguration configuration)
+        {
+            ConfigureMySQLContext(services, configuration);
```
Comment (romango78) [inline Taxopark/DependencyInjection/DependencyContainer.cs]: The more generic name (ex. ConfigureDbContext) can be used when you don't expect to use several databases.

File: Taxopark/Taxopark/appsettings.json [hunk 2]
```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "mysqlconnection": {
+    "connectionString": "server=db;port=3306;userid=dbuser;password=dbuserpassword;database=taxopark;"
```
Comment (romango78) [inline Taxopark/Taxopark/appsettings.json]: It will be more secure when password will not be stored in any settings/config files.

**Review by romango78 — APPROVED** [review]
See inline comments.

File: Taxopark/DependencyInjection/DependencyContainer.cs (hunk 1 above)
Comment (natatkachenko) [reply Taxopark/DependencyInjection/DependencyContainer.cs]: Good point, the method name was changed to ConfigureDBContext.

**Comment by natatkachenko** [comment]
A hack was implemented to avoid the error "Invalid version: 15.2-15.3" which caused the build of the Angular application to ruin. 
Commit [3eedf3c](https://github.com/romango78/LP-IS-Taxopark/pull/15/commits/3eedf3c95e64e2b34616d4d9497338df9d8a099f)
A discussion about this bug can be found here: https://github.com/nrwl/nx/issues/8768

File: Taxopark/Taxopark/appsettings.json (hunk 2 above)
Comment (natatkachenko) [reply Taxopark/Taxopark/appsettings.json]: A user password was removed.

**Pull request merged**